extensions/wikia/CreateNewWiki/SpecialCreateNewWiki.class.php

```diff
@@ -11,10 +11,16 @@ public function __construct() {
 	 * @throws ErrorPageError
 	 */
 	public function execute( $par ) {
+		global $wgUCPCommunityCNWAddress;
+
 		wfProfileIn( __METHOD__ );
 		$out = $this->getOutput();
 		$user = $this->getUser();
 
+		if ( !$user->isStaff() ) {
+			$out->redirect( WikiFactory::getLocalEnvURL( $wgUCPCommunityCNWAddress ) );
```

Comment:
This might work on prod but won't probably work on dev since there  is different wiki. We need to sync those after we switch